CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+## v1.1.0 (2024-03-05)
+
+[Detailed release notes](https://github.com/cytomining/pycytominer/releases/tag/v1.1.0)
+
+### Fix
+
+- **build**: fix build versioning
+- simplify Spherize transform – epsilon to regularize instead of clip, add additional checks (see #320)
+
+### Refactor
+
+- **docs**: apply flake8-builtins checks
+- **dev**: apply pyflakes checks
+- **dev**: apply flake8-simplify checks
+
+### Test
+
+- add flake8-bandit ignores
+- add clarifying comments for cell_loc test
+
```

Comment:
We should add a note here about your testing changes prior to merging.

Reply:
Great point, I'll see that this is added in subsequent changes for this PR.